### PR TITLE
Move in all kernel config fragments

### PR DESCRIPTION
**Pull request by matinlotfali**
@gfinlay Please add your kernel configs in this branch.

### CODE REVIEW

File: recipes-kernel/linux/Kconfigs/ATH6KL.cfg [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# CONFIG_PRISM54 is not set
+CONFIG_WLAN_VENDOR_MARVELL=y
+CONFIG_WLAN_VENDOR_MEDIATEK=y
+CONFIG_WLAN_VENDOR_RALINK=y
+CONFIG_WLAN_VENDOR_REALTEK=y
+CONFIG_WLAN_VENDOR_RSI=y
+CONFIG_WLAN_VENDOR_ST=y
+CONFIG_WLAN_VENDOR_TI=y
+CONFIG_WLAN_VENDOR_ZYDAS=y
+CONFIG_WLAN_VENDOR_QUANTENNA=y
```
Comment (mistyron) [inline recipes-kernel/linux/Kconfigs/ATH6KL.cfg:22]: Is this still needed? I thought this was removed at some point. I think I added this initially for the laird but by mistake and it's not actually needed for the LWB5p module, please confirm

**Review by mistyron — COMMENTED** [review]
confirm if ATH6KL.cfg is needed

File: recipes-kernel/linux/Kconfigs/ATH6KL.cfg (hunk 1 above)
Comment (matinlotfali) [reply recipes-kernel/linux/Kconfigs/ATH6KL.cfg:22]: I believe it is not needed. Should I delete it?

Comment (mistyron) [reply recipes-kernel/linux/Kconfigs/ATH6KL.cfg:22]: If it's not needed, yes, delete it

**Comment by matinlotfali** [comment]
@gfinlay Can you also add your files to `linux-renesas_%.bbappend`?

**Comment by mistyron** [comment]
@vlaltzic you should probably add your fragment here too, please sync with @matinlotfali

**Comment by gfinlay** [comment]
> @gfinlay Can you also add your files to `linux-renesas_%.bbappend`?

Done.

**Review by mistyron — APPROVED** [review]
LGTM

**Review by gfinlay — APPROVED** [review]
Approved.

**Pull request merged**